README.md

```diff
@@ -19,7 +19,7 @@ for more details.
 
 Make sure you have the following installed:
 
-- [Node.js](https://nodejs.org/en/download/) >= 7.0.0
+- [Node.js](https://nodejs.org/en/download/) >= 8.0.0
```

Comment:
Technically speaking, we support Node.js 6.x too, our CI is configured to run the tests on this platform too.

The top-level `package.json` in this github (mono)repo says so too:

https://github.com/strongloop/loopback-next/blob/051b8e0e86045b20afa991bb120f01380b62c01b/package.json#L8-L10

I checked `package.json` files in individual packages (that are eventually published to npmjs.org) and apparently they are missing the required `engines` field. I opened https://github.com/strongloop/loopback-next/pull/609 to fix that.

I am proposing to change this line to say `Node.js 6.x, 8.x or newer`. Dropping support for Node.js 6.x requires changes in many places, I strongly prefer to make such change in a standalone pull request.

Reply:
@bajtos , since we'll need confirmation from Savio on the node version support, I'd revert my change on the version for now.